cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -174,6 +175,27 @@ def _read_old_file_lines(self, file_path):
             lines = f.readlines()
         return lines
 
+    def _bytes_as_mb(self, value_in_bytes):
+        return '{}MB'.format(value_in_bytes / 1024 / 1024)
+
+    def _generate_default_shared_buffers(self):
+        return self._bytes_as_mb(psutil.virtual_memory().total / 4)
```

Comment:
might as well be explicit that you want integer division, and do `//`
(python 3 would make the `x / 4` float division, and this would give float)

(ditto for all other uses of `/`)